templates/docker-compose/docker-compose.yml.dist

```diff
@@ -7,14 +7,17 @@ services:
         restart: always
         depends_on:
             - supportpal
+            - supportpal_websockets
         environment:
             WEB_CONTAINER: ${WEB_SERVICE_NAME}
+            WS_SERVICE_NAME: ${WS_SERVICE_NAME}
         volumes:
             - ./gateway/nginx-http.conf:/etc/nginx/templates/default.conf.template
             - ./gateway/base.conf:/etc/nginx/templates/base.config.template
             - ./gateway/custom.conf:/etc/nginx/templates/custom.config.template
         ports:
             - '${HTTP_REMOTE_PORT}:80'
+            - '${SOCKETS_PORT}:6001'
```

Comment:
I don't think this is correct, we only want to expose `http` and `https`. It should be using https://beyondco.de/docs/laravel-websockets/basic-usage/ssl#same-location-for-websockets-and-web-contents

Reply:
Also if we proxy connections from nginx to the web socket server via http, is that going to make an insecure external network connection in AWS EB?

Reply:
will fix the others according to the reference.  in AWS EB, (mainly HA) it will still be secure, since everything is behind the load balancer

Reply:
Otherwise, it's inside the internal AWS network which should be secure even with HTTP